packages/awslint/tsconfig.json

```diff
@@ -11,6 +11,7 @@
       "noUnusedLocals": true,
       "noUnusedParameters": true,
       "noImplicitReturns": true,
+      "experimentalDecorators": true,
```

Comment:
Note for posterity / other people who come across this PR and wonder:

This shouldn't be necessary unless you install `jsii-reflect` from a local source tree (which I assume is what you did to validate this PR) without adding a project reference in `tsconfig.json` (because then, `tsc` will read the `.ts` files which contain a decorator -- `@memoized` -- instead of the `.d.ts` files where the decorator is stripped).